diff --git a/loaders/HDRCubeTextureLoader.d.ts b/loaders/HDRCubeTextureLoader.d.ts
index 800154217d184b027dca7bb6b6222b77984e7a32..e08f55e2c4c9c0b993d68e9be84f115dae17b06a 100644
--- a/loaders/HDRCubeTextureLoader.d.ts
+++ b/loaders/HDRCubeTextureLoader.d.ts
@@ -8,7 +8,7 @@ export class HDRCubeTextureLoader extends Loader {
   type: TextureDataType
 
   load(
-    urls: string[],
+    urls: string | string[],
     onLoad: (texture: CubeTexture) => void,
     onProgress?: (event: ProgressEvent) => void,
     onError?: (event: ErrorEvent) => void,
